index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>User Form Submit</title>
+</head>
+
+<body>
+    <form>
+        <label for="firstName">First Name</label>
+        <input type="text" name="firstName" id="firstName">
+        <label for="lastName">Last Name</label>
+        <input type="text" name="lastName" id="lastName">
+        <label for="age">Age</label>
+        <input onkeyup="validateData('age')" type="number" name="age" id="age">
+        <label for="email">Email</label>
+        <input type="email" name="email" id="email">
+        <label for="phone">Phone Number : </label>
+        <input onkeyup="validateData('phone')" type="number" name="phone" id="phone">
+        <p class="error"> </p>
+        <button id="btn">Submit</button>
```

Comment:
@Obliger0 Can we change `button` id to `submitButton`?  for  better understanding